pyproject.toml

```diff
@@ -12,8 +12,55 @@ addopts = [
 omit = [".env/*"]
 sort = "Cover"
 
-#[report]
-#sort = Cover
-#omit =
-#    .env/*
-#    backtracking/*
+[tool.codespell]
+ignore-words-list = "3rt,ans,crate,damon,fo,followings,hist,iff,kwanza,mater,secant,som,sur,tim,zar"
+skip = "./.*,*.json,ciphers/prehistoric_men.txt,project_euler/problem_022/p022_names.txt,pyproject.toml,strings/dictionary.txt,strings/words.txt"
+
+[tool.ruff]
+ignore = [
+    "B904",
+    "B905",
+    "E741",
+    "N999",
+    "PLC1901",
+    "PLR2004",
+    "PLR5501",
+    "PLW0120",
+    "PLW060",
+    "PLW2901",
+    "RUF00",
+    "RUF100",
+    "S101",
+    "S105",
+    "S113",
+    "UP038",
+]
+select = [
```

Comment:
A brief description for the selected rules will be helpful. What do you think?

Reply:
Done.